assets/javascripts/discourse/components/endorsement-button.gjs

```diff
@@ -81,7 +81,7 @@ export default class EndorsementButton extends Component {
           <DButton
             @action={{this.openEndorsementModal}}
             class="btn-flat category-expert-endorse-edit"
```

Comment:
should the class be updated?

Reply:
or btn-flat don't need it?

Reply:
No, I think that's fine. I'm mainly replacing buttons without any descriptive class. This one has btn-flat, so that can be okay.